src/main/java/com/mycompany/myapp/generator/server/springboot/banner/domain/infrastructure/primary/BannerModuleExtensionConfiguration.java

```diff
@@ -0,0 +1,82 @@
+package com.mycompany.myapp.generator.server.springboot.banner.domain.infrastructure.primary;
```

Comment:
There's still an issue, the `infrastructure` and `application` packages should not be placed inside the `domain` package
```suggestion
package com.mycompany.myapp.generator.server.springboot.banner.infrastructure.primary;
```

Reply:
Thanks, @murdos. Silly mistake 😅